source/includes/aws/_buckets.md

```diff
@@ -47,7 +47,44 @@ Retrieve a list of all buckets from Amazon S3.
 | `name`<br/>*string*               | The name of the bucket.                                                                                                                                                                                                |
 | `region`<br/>*string*     | The region the bucket exists in.                                                                                                                                                     |
 | `created`<br/>*string*            | The date the bucket was created.                                                                                                                                                                         |
-| `url`<br/>*string*       | The full endpoint url used to make api calls on the bucket.                                                                                                                                                                                                                                               
+| `url`<br/>*string*       | The full endpoint url used to make api calls on the bucket.                                                                                                                                                                                                                     
+
+<!-------------------- Update buckets -------------------->
+
+#### Update Bucket Permissions
+
+```shell
+curl -X POST \
```

Comment:
Are update requests generally POST? Was under the impression they were PUT.

Reply:
For plugin, the operation are all POST and you specify the operation name. (From my understanding)
Here's an example of an update operation that also uses POST.
![Screen Shot 2022-03-08 at 10 59 18 AM](https://user-images.githubusercontent.com/35710207/157277274-efa6bc1a-84c3-435d-9ac8-b452e16eac26.png)

Reply:
I check the code and indeed it is a post, but if did put the default for update, you could have use the put 
![image](https://user-images.githubusercontent.com/56133634/157526161-d4c4ee6a-2d68-4ab6-95aa-83b9378b170a.png)